ymir/command/mir/commands/evaluate.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import logging
+from typing import List
+
+import numpy as np
+
+from mir.commands import base
+from mir.tools import checker, eval, mir_storage_ops, revs_parser
+from mir.tools.code import MirCode
+from mir.tools.command_run_in_out import command_run_in_out
+from mir.tools.errors import MirRuntimeError
+from mir.protos import mir_command_pb2 as mirpb
+
+
+class CmdEvaluate(base.BaseCommand):
+    def run(self) -> int:
+        logging.debug("command evaluate: %s", self.args)
+
+        return CmdEvaluate.run_with_args(work_dir=self.args.work_dir,
+                                         src_revs=self.args.src_revs,
+                                         dst_rev=self.args.dst_rev,
+                                         gt_rev=self.args.gt_rev,
+                                         mir_root=self.args.mir_root,
+                                         conf_thr=self.args.conf_thr,
+                                         iou_thrs=self.args.iou_thrs)
+
+    @staticmethod
+    @command_run_in_out
+    def run_with_args(work_dir: str, src_revs: str, dst_rev: str, gt_rev: str, mir_root: str, conf_thr: float,
+                      iou_thrs: str) -> int:
+        src_rev_tids = revs_parser.parse_arg_revs(src_revs)
+        gt_rev_tid = revs_parser.parse_single_arg_rev(gt_rev, need_tid=False)
+        dst_rev_tid = revs_parser.parse_single_arg_rev(dst_rev, need_tid=True)
+
+        return_code = checker.check(mir_root,
+                                    [checker.Prerequisites.IS_INSIDE_MIR_REPO, checker.Prerequisites.IS_CLEAN])
+        if return_code != MirCode.RC_OK:
+            return return_code
+
+        iou_thrs_list = [float(v) for v in iou_thrs.split(':')]
+        iou_thr_from, iou_thr_to, iou_thr_step = iou_thrs_list
```

Comment:
iou_thr_from, iou_thr_to, iou_thr_step = [float(v) for v in iou_thrs.split(':')]

Reply:
update